src/zope/app/generations/demo/__init__.py

```diff
@@ -0,0 +1,29 @@
+##############################################################################
+#
+# Copyright (c) 2004 Zope Foundation and Contributors.
+# All Rights Reserved.
+#
+# This software is subject to the provisions of the Zope Public License,
+# Version 2.1 (ZPL).  A copy of the ZPL should accompany this distribution.
+# THIS SOFTWARE IS PROVIDED "AS IS" AND ANY AND ALL EXPRESS OR IMPLIED
+# WARRANTIES ARE DISCLAIMED, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF TITLE, MERCHANTABILITY, AGAINST INFRINGEMENT, AND FITNESS
+# FOR A PARTICULAR PURPOSE.
+#
+##############################################################################
+"""Demo package for evolution scripts
+
+The evolution scripts in this package are pretty dumb. The just call
+the evolve function defined here with a generation number.
+
+"""
+key = 'zope.generations.demo-generation'
```

Comment:
I wonder if this should also be `zope.app.generations.demo-generation`?

Reply:
Of course. Thank you for catching it.